packages/tempus/src/Tempus_IntegratorForwardSensitivity_decl.hpp

```diff
@@ -237,39 +228,44 @@ class IntegratorForwardSensitivity
   //@}
 
 protected:
-
-  // Create sensitivity model evaluator from application model
-  void
-  createSensitivityModelAndStepper(
-    const Teuchos::RCP<Thyra::ModelEvaluator<Scalar> >& model);
-
-  Teuchos::RCP<Thyra::ModelEvaluator<Scalar> > model_;
-  Teuchos::RCP<SensitivityModelEvaluatorBase<Scalar> > sens_model_;
-  Teuchos::RCP<StepperStaggeredForwardSensitivity<Scalar> > sens_stepper_;
-  Teuchos::RCP<IntegratorBasicOld<Scalar> > integrator_;
-  Teuchos::RCP<Teuchos::ParameterList> tempus_pl_;
-  Teuchos::RCP<Teuchos::ParameterList> sens_pl_;
-  Teuchos::RCP<Teuchos::ParameterList> stepper_pl_;
+  Teuchos::RCP<Thyra::ModelEvaluator<Scalar>> model_;
+  Teuchos::RCP<IntegratorBasic<Scalar>> integrator_;
+  Teuchos::RCP<SensitivityModelEvaluatorBase<Scalar>> sens_model_;
+  Teuchos::RCP<StepperStaggeredForwardSensitivity<Scalar>> sens_stepper_;
   bool use_combined_method_;
 };
 
 /// Nonmember constructor
-template<class Scalar>
-Teuchos::RCP<IntegratorForwardSensitivity<Scalar> >
-integratorForwardSensitivity(
-  Teuchos::RCP<Teuchos::ParameterList>                pList,
-  const Teuchos::RCP<Thyra::ModelEvaluator<Scalar> >& model);
-
-/// Nonmember constructor
-template<class Scalar>
-Teuchos::RCP<IntegratorForwardSensitivity<Scalar> >
+/**
+ * @brief Nonmember constructor
+ *
+ * This nonmember constructor calls parses the `pList` provided to constructor
+ * the sub-objects needed to call the full `IntegratorForwardSensitivity`
+ * construtor
+ *
+ * @param pList  ParameterList defining the integrator options and options
+ *               defining the sensitivity analysis
+ * @param model  ModelEvaluator for the problem
+ *
+ * @return Time integrator implementing forward sensitivity
+ */
+template <class Scalar>
+Teuchos::RCP<IntegratorForwardSensitivity<Scalar>>
 integratorForwardSensitivity(
```

Comment:
We have gone with the convention of prepending "create" to nonmember constructors and use camel case, e.g., integratorForwardSensitivity()  --> createIntegratorForwardSensitivity() to indicate these are nonmember constructors.